### ISSUE TITLE
Loadout Optimizer “Pin Item” selection dialog not appearing on iPad

### ISSUE THREAD

**Issue by jbozanowski** [DestinyItemManager/DIM#7111]
After clicking "Pin item" button I'm getting a grey background where the item selection dialog should be and nothing happens until I open the main menu in upper left corner. Then the dialog appears and then everything works as expected.

Attaching screen recording, at the beginning I'm scrolling a bit to show that scrolling works now (there is a separate issue about that, I'm going to leave a comment with the same recording there as well).

I'm on iPad Air 4th gen, using DIM beta (updated via the menu just before starting the recording), launched from a desktop shortcut, so it appears without any browser chrome (in case this matters, too).

https://imgur.com/a/pBbG7k7

**Comment by bhollis** [comment]
What iOS version?

**Comment by jbozanowski** [comment]
Ah, of course, sorry. 15.0.1.
